<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProviderDialog</class>
 <widget class="QDialog" name="AddProviderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add provider</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="txtName"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblEmail">
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLineEdit" name="txtEmail"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblPhone">
     <property name="text">
      <string>Phone:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLineEdit" name="txtPhone"/>
   </item>
   <item row="8" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>236</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="btnOk">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblCompany">
     <property name="text">
      <string>Company:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QLineEdit" name="txtCompany"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtEmail</tabstop>
  <tabstop>txtPhone</tabstop>
  <tabstop>txtCompany</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>AddProviderDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>AddProviderDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
